### PR TITLE
feat(be): implement token obtain pair (login) and register endopints 

### PR DESCRIPTION
**Pull request by MucahitErdoganUnlu**
* Bootstrapped the project and the app #513 
* Implemented login and register endpoints #516 
* Documented these endpoints on swagger #515 

Please be careful to select *Squash and merge* while merging as @umitcan07 suggested

### CODE REVIEW

File: backend/README.md
Comment (meminciftci) [inline backend/README.md:1]: Before running, execute " $ python manage.py migrate".

File: backend/core/tests.py
Comment (meminciftci) [inline backend/core/tests.py:1]: Remove this file

**Review by meminciftci — CHANGES_REQUESTED** [review]
Change the commented fields

**Review by ArdaVural — APPROVED** [review]

**Comment by ArdaVural** [comment]
I adressed these issues.

> Change the commented fields

**Pull request merged**